src/main/java/org/jenkinsci/plugins/rundeck/RundeckCause.java

```diff
@@ -158,7 +169,7 @@ public String getIconFileName() {
         }
 
         public String getUrlName() {
-            return execution != null ? execution.getUrl() : null;
+            return execution != null ? execution.getHref() : null;
```

Comment:
I think it should use execution.getPermalink(), which is the link to the rundeck GUI.  the HREF is the link to the API